processing/src/main/java/io/druid/query/filter/DruidLongPredicate.java

```diff
@@ -28,5 +28,25 @@
 
```

Comment:
It's strange that you had to add `@SuppressWarnings` in Double and Float variants but not here.

Reply:
For Long variant there are still implementations that are not lambda expressions, so intellij was not complaining about unused variable.